lib/common_test/src/ct_run.erl

```diff
@@ -131,8 +132,9 @@ script_start(Args) ->
 			    _ -> ""
 			end
 		end,
-	    io:format("~nCommon Test~s starting (cwd is ~ts)~n~n",
-	              [CTVsn,Cwd]),
+            Header = io_lib:format("common test~s starting", [CTVsn]),
```

Comment:
`Common Test` spelling is all over the place (code and docs). Should we leave it with capitals?